.travis.yml

```diff
@@ -5,26 +5,6 @@ git:
 
 matrix:
   include:
-    - php: 5.4
```

Comment:
Sorry, I thought you wanted to remove that because of PHP support end. @kylekatarnls

Reply:
End of support version does not mean PHP will suddently stop working if you use PHP 5.4 and if we don't test it we won't know if we're still compatible with it (or rather we will probably miss incompatibilites). So before any PHP version support drop in the code or in testing environnent, any PHP software should first increase the minimum requirement in its composer.json to reflect the reality, then bump our own version number (at least a minor bump). Then I will not remove PHP 5 current range as long as we don't need a change big enough to rewrite the whole thing up to the PHP 7 standards.